tests/config/test_config.py

```diff
@@ -0,0 +1,95 @@
+"""Test config."""
+
+import os
+import tempfile
+
+import pytest
+from pydantic import ValidationError
+
+from ocf_data_sampler.config.load import load_yaml_configuration
+from ocf_data_sampler.config.model import Configuration
+from ocf_data_sampler.config.save import save_yaml_configuration
+
+
+def test_default():
+    """
+    Test default pydantic class
+    """
+
+    _ = Configuration()
+
+
+def test_yaml_load_test_config(test_config_filename):
+    """Test that yaml loading works for 'test_config.yaml'"""
+
+    config = load_yaml_configuration(test_config_filename)
+
+    assert isinstance(config, Configuration)
+
+
+def test_yaml_save():
+    """
+    Check a configuration can be saved to a yaml file
+    """
+
+    with tempfile.NamedTemporaryFile(suffix=".yaml") as fp:
+        filename = fp.name
+
+        # check that temp file can't be loaded
+        with pytest.raises(TypeError):
+            _ = load_yaml_configuration(filename)
+
+        # save default config to file
+        save_yaml_configuration(Configuration(), filename)
+
+        # check the file can be loaded
+        _ = load_yaml_configuration(filename)
+
+
+def test_yaml_load_env(test_config_filename):
+    """
+    Check a configuration can be loaded with an env var
```

Comment:
I don't think the functionality to load from environmental variable exists - I also don't think it should. I'd remove this

Reply:
yea, I think we used to use it, but not any more. It was a nice feature meaing you didnt have to edit the config file with your ENV VARS, but to keep things simple, can strip this out

Reply:
Okay, well since that functionality doesn't exist anymore I think we should get rid of it for now